<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>620</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gestión de Sueldos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: rgb(0, 0, 0);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>651</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 40pt &quot;Calibri&quot;;
color: rgb(182, 169, 89);
</string>
    </property>
    <property name="text">
     <string>GESTIONAR TRABAJADOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BuscarExistente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 25pt &quot;Calibri&quot;;
color: white;
border: 1px solid white;
background-color: rgb(41, 11, 11);</string>
    </property>
    <property name="text">
     <string>Buscar existente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RegistrarNuevo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>25</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 25pt &quot;Calibri&quot;;
color: white;
border: 1px solid white;
background-color: rgb(41, 11, 11);</string>
    </property>
    <property name="text">
     <string>Registrar nuevo</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRegresar">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>500</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>flecha_regresar.png</normaloff>flecha_regresar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>darkTrabajadorMejorado.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_BuscarExistente</zorder>
   <zorder>pushButton_RegistrarNuevo</zorder>
   <zorder>pushButtonRegresar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubarMenuTrabajador">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="menubarArchivo">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menubarMenu">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Menú</string>
    </property>
    <addaction name="actionInicio"/>
    <addaction name="actionTrabajador"/>
    <addaction name="actionBonificaciones"/>
   </widget>
   <widget class="QMenu" name="menubarDetallesDelSoftware">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Detalles del Software</string>
    </property>
    <addaction name="actionAcercaDeGestorDeSueldosV1"/>
    <addaction name="actionAcercaDeNosotros"/>
   </widget>
   <addaction name="menubarArchivo"/>
   <addaction name="menubarMenu"/>
   <addaction name="menubarDetallesDelSoftware"/>
  </widget>
  <action name="actionSobre_nosotros">
   <property name="text">
    <string>Sobre nosotros</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAa">
   <property name="text">
    <string>Aa</string>
   </property>
  </action>
  <action name="actionInicio">
   <property name="text">
    <string>Inicio</string>
   </property>
  </action>
  <action name="actionTrabajador">
   <property name="text">
    <string>Trabajador</string>
   </property>
  </action>
  <action name="actionBonificaciones">
   <property name="text">
    <string>Bonificaciones</string>
   </property>
  </action>
  <action name="actionAcercaDeGestorDeSueldosV1">
   <property name="text">
    <string>Acerca de Gestor de Sueldos v1</string>
   </property>
  </action>
  <action name="actionAcercaDeNosotros">
   <property name="text">
    <string>Acerca de nosotros</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
